### ISSUE TITLE
A few issues

### ISSUE THREAD

**Issue by rallisf1** [ugoviti/izpbx#27]
Hello, please consider the following:

* Adding ports 8089, 8001, 8003 to the mapping rules as those are used by the UCP node server and WebRTC.
* Expose /etc/fail2ban so we can add client IPs to the ignore list later on
* Auto-Update FreePBX and its modules on first run. Also add a FREEPBX_UPDATE flag in the env so people can enable/disable it.

And a couple questions:

1. Can I use external SMTP without running postfix at all? I prefer it than enable relaying on my main mail server.
2. How can I change hostname (APP_FQDN) after installation?

**Comment by ugoviti** [comment]
Hi John,

thank you for your feedback.. 

- new ports support are already in the main branch
- you can whitelist IP using the `FAIL2BAN_DEFAULT_IGNOREIP` variable, this isn't sufficient?
- good suggestion... staged for the next release (releasing today as 18.15.24)

About your questions:

1. nope, postfix is used as local MTA, but you can configure remote smarthost on local postfix changing these variables:
```
SMTP_RELAYHOST=[smtp.example.com]:587
SMTP_RELAYHOST_USERNAME=yourusername
SMTP_RELAYHOST_PASSWORD=yoursecurepassword
SMTP_STARTTLS=true
```
2. if you change the APP_FQDN variable in the .env that doesn't works?

Kind Regards

**Comment by rallisf1** [comment]
Hey,

I haven't tried changing the env variables and recreating the container, I thought it would break something. I'll try that (both for fail2ban conf and APP_FQDN) and let you know.

Grazie Mille!

**Comment by ugoviti** [comment]
released 18.15.24

https://github.com/ugoviti/izdock-izpbx/blob/main/CHANGELOG.md

feel free to close this issue after tested the release

Grazie a te! ;)

**Comment by rallisf1** [comment]
Forgot to close this :>

Everything works as intented, bravo signore!
